src/main/kotlin/com/coder/gateway/CoderRemoteConnectionHandle.kt

```diff
@@ -523,5 +534,26 @@ class CoderRemoteConnectionHandle {
 
     companion object {
         val logger = Logger.getInstance(CoderRemoteConnectionHandle::class.java.simpleName)
+        @Throws(CoderSetupCommandException::class)
+        fun processSetupCommand(
+            ignoreSetupFailure: Boolean,
```

Comment:
Not a comment on this PR just a note to self about how we're doing this in general: don't love us drilling down this `Boolean` flag for ignoring errors.